repository/src/main/resources/alfresco/encryption-context.xml

```diff
@@ -126,4 +126,5 @@
     <bean id="keyStoreChecker" class="org.alfresco.encryption.KeyStoreChecker">
         <property name="mainKeyStore" ref="keyStore"/>
     </bean>
+

```

Comment:
Doesn't really make a difference, but removing that could remove the file mention in this PR. I don't mind it either way though